new-lamassu-admin/src/pages/UserManagement/modals/ChangeRoleModal.js

```diff
@@ -0,0 +1,67 @@
+import { makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+
+import Modal from 'src/components/Modal'
+import { Button } from 'src/components/buttons'
+import { Info2, P } from 'src/components/typography'
+
+import styles from '../UserManagement.styles'
+
+const useStyles = makeStyles(styles)
+
+const ChangeRoleModal = ({
+  showModal,
+  toggleModal,
+  user,
+  confirm,
+  inputConfirmToggle,
+  setAction
+}) => {
+  const classes = useStyles()
+
+  const handleClose = () => {
+    toggleModal()
+  }
+
+  return (
+    <>
+      {showModal && (
+        <Modal
+          closeOnBackdropClick={true}
+          width={450}
+          height={250}
+          handleClose={handleClose}
+          open={true}>
```

Comment:
if the modal already has the prop open={something}, why do we need the explicit` showModal && ` check?
this applies to the other modals as well

Reply:
Just spoke to Rafael and tested it myself. As the modals require outside information to process its logic, it is not possible to change to `open={showModal}` as it will break the page it's in